### PR TITLE
[10.0] partner multi relation

### PR DESCRIPTION
**Pull request by NL66278**

### CODE REVIEW

**Comment by gurneyalex** [comment]
hello @NL66278 

I just submitted https://github.com/NL66278/partner-contact/pull/1 to fix travis for your PR.

**Comment by leemannd** [comment]
Hello @NL66278 
I've submitted also https://github.com/NL66278/partner-contact/pull/3 to have more consistency in the views and avoid a 'README' conflict

**Comment by sharkende** [comment]
Hello,
I would like to know if the PR is ready to merge ?

**Comment by NL66278** [comment]
@sharkende I think it is ready to merge.

**Review by chienandalu — APPROVED** [review]
Functionality and code check :+1:

**Comment by rafaelbn** [comment]
Hi, thanks. It's OK. Just if possible could you reorder menu (first Customer second Relations)

**Review by rafaelbn — APPROVED** [review]
tested, It' OK just https://github.com/OCA/partner-contact/pull/363#issuecomment-304660569

File: .travis.yml [hunk 1]
```diff
@@ -22,8 +22,10 @@ env:
   matrix:
   - LINT_CHECK="1"
   - TRANSIFEX="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
+  - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE=partner_multi_relation
```
Comment (pedrobaeza) [inline .travis.yml]: Why do we separate it?

File: README.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ addon | version | summary
 [partner_firstname](partner_firstname/) | 10.0.2.0.0 | Split first name and last name for non company partners
 [partner_helper](partner_helper/) | 10.0.0.1.0 | Add specific helper methods
 [partner_identification](partner_identification/) | 10.0.1.0.1 | Partner Identification Numbers
+[partner_multi_relation](partner_multi_relation/) | 10.0.1.0.0 | Partner Relations
```
Comment (pedrobaeza) [inline README.md]: Don't modify this file. It's done automatically and probably provoke conflicts

File: .travis.yml (hunk 1 above)
Comment (NL66278) [reply .travis.yml]: @pedrobaeza This was proposed by @gurneyalex but I cannot find back why. Probably no longer needed.

Comment (pedrobaeza) [reply .travis.yml]: Oh, I remember. It was a way to avoid some kind of side effects on other modules tests. Let's remove it and see if it works

Comment (hbrunn) [reply .travis.yml]: in earlier versions, we had some wild patches on crucial methods (res.partner#write for example) which triggered some bugs and also had some bugs itself

Comment (pedrobaeza) [reply .travis.yml]: So do you think it's still needed?

**Comment by pedrobaeza** [comment]
Please squash all the "OCA Transbot" commits together

File: .travis.yml (hunk 1 above)
Comment (hbrunn) [reply .travis.yml]: no clue, I didn't look into this version's code yet. But I wanted to give the rationale why it used to be tested separately

Comment (pedrobaeza) [reply .travis.yml]: OK, thanks for the information. The only problem with this technique is that runbot only installs the first declared module (in this case, partner_multi_relation), and not the rest. If we continue splitting this, better to put first the EXCLUDE line to have all the modules except this one.

**Comment by rafaelbn** [comment]
@NL66278 this PR is Ready to Merge, as said "Please squash all the "OCA Transbot" commits together" thanks!

**Comment by yajo** [comment]
It's not ready, travis is ❌

**Comment by NL66278** [comment]
I removed the commits that in some way had wormed their way into this branch, but were not related at all.

**Comment by NL66278** [comment]
Travis error caused by travis itself apparently running not to completion. Nothing to do with this module. Unfortunately due to travis error latest commits have not been tested at all by Travis.

**Comment by yajo** [comment]
Yeah, it seems to be https://github.com/OCA/maintainer-quality-tools/issues/448; let's wait until it gets fixed.

**Comment by pedrobaeza** [comment]
Please rebase to get a green branch

**Comment by NL66278** [comment]
@pedrobaeza @Yajo rebase done and everything is green now.

**Pull request merged**